pulp_container/tests/functional/api/test_build_images.py

```diff
@@ -61,3 +63,45 @@ def test_build_image(
     local_registry.pull(distribution.base_path)
     image = local_registry.inspect(distribution.base_path)
     assert image[0]["Config"]["Cmd"] == ["cat", "/tmp/inside-image.txt"]
+
+
+def test_build_image_from_repo_version(
+    containerfile_name,
+    container_distribution_api,
+    container_repository_api,
+    file_bindings,
+    file_content_unit_with_name_factory,
+    file_repo_with_auto_publish,
+    gen_object_with_cleanup,
+    local_registry,
+):
+    """Test if a user can build an OCI image."""
+    repository = gen_object_with_cleanup(
+        container_repository_api, ContainerContainerRepository(**gen_repo())
+    )
+
+    files = {
+        "txt": file_content_unit_with_name_factory("foo/bar/example.txt"),
+    }
+
+    units_to_add = list(map(lambda f: f.pulp_href, files.values()))
+    data = RepositoryAddRemoveContent(add_content_units=units_to_add)
+    monitor_task(
+        file_bindings.RepositoriesFileApi.modify(file_repo_with_auto_publish.pulp_href, data).task
+    )
+
+    build_response = container_repository_api.build_image(
```

Comment:
Do you mind adding a test case that triggers users' permissions checks before building an image?

```
with user_helpless, pytest.raises(APIException):
    container_repository_api.build_image()
    
with user_manager:
    container_repository_api.build_image()

...
```